mesa-quilt

```diff
@@ -18,17 +18,73 @@
 # 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
 d=. ; while [ ! -d "$d/nx-X11/extras" ] && [ "$(readlink -e "$d")" != "/" ]; do d="$d/.."; done
-if [ -d "$d/nx-X11/extras" ] && [ -z "$QUILT_PATCHES" ]; then
-    # if in nx-libs tree with unset $QUILT_PATCHES
-    export QUILT_PATCHES="../Mesa.patches"
-    export QUILT_PATCH_OPTS="--reject-format=unified"
-    export QUILT_DIFF_ARGS="-p ab --no-timestamps --no-index --color=auto"
-    export QUILT_REFRESH_ARGS="-p ab --no-timestamps --no-index"
-    export QUILT_COLORS="diff_hdr=1;32:diff_add=1;34:diff_rem=1;31:diff_hunk=1;33:diff_ctx=35:diff_cctx=33"
-    if ! [ -d "$d/nx-X11/extras/Mesa.patches" ]; then mkdir "$d/nx-X11/extras/Mesa.patches"; fi
-    cd "$d/nx-X11/extras/Mesa/"
-fi
 
-quilt "$@"
+if [ -d "$d/nx-X11/extras" ]; then
+
+	if which quilt 1>/dev/null; then
```

Comment:
Oh, on the contrary, that's perfectly POSIX-compliant behavior:

>[2.7.2 Redirecting Output](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18)
>
>The two general formats for redirecting output are:
>
>[n]>word
>[n]>|word
>
>where the optional n represents the file descriptor number. If the number is omitted, the redirection shall refer to standard output (file descriptor 1).

If dash failed to implement that correctly (and yet claimed compliance), then it was broken.